site/docs/v1/tech/guides/user-search-with-elasticsearch.adoc

```diff
@@ -74,6 +74,10 @@ There are three parameters for a search, and they are mutually exclusive.
 | Yes
 | Useful to leverage the full power of Elasticsearch queries and need to look across different fields, including nested values, and/or use compound or complex search parameters such as ranges.
 
+| `nextResults`
+| Yes
+| Allows you to continue paging forward in the results set after an initial `query` or `queryString` query. Useful for paging past the 10000 results limit in Elasticsearch. See link:docs/v1/tech/reference/limitations#user-searches[Limitations] for more information. Available since version 1.48.0.
```

Comment:
```suggestion
| Allows you to continue paging forward in the results set after an initial `query` or `queryString` query. Useful for paging past the 10000 results limit in Elasticsearch. See link:/docs/v1/tech/reference/limitations#user-searches[Limitations] for more information. Available since version 1.48.0.
```